tf2onnx/onnx_opset/logical.py

```diff
@@ -46,30 +41,56 @@ def version_4(cls, ctx, node, **kwargs):
         pass
 
 
-@tf_op(["Equal", "Greater", "Less"])
 @tf_op("LogicalAnd", onnx_op="And")
 @tf_op("LogicalOr", onnx_op="Or")
 class BroadcastOp(common.BroadcastOp):
     pass
 
 
+@tf_op("Equal")
+class Equal:
+    @classmethod
+    def version_4(cls, ctx, node, **kwargs):
+        common.BroadcastOp.version_4(ctx, node, **kwargs)
+
+    @classmethod
+    def version_7(cls, ctx, node, **kwargs):
+        # T2 output = Equal(T1, x, T1 y), T1 \in {bool, int32, int64}
```

Comment:
```\```?

Reply:
aha \in means ![in_equation](http://mathurl.com/render.cgi?%5Cinlinemode%20%5Cin%5Cnocache) in math equations.

Reply:
learned new things. :-)